app/src/main/java/com/cris/nvh/framgiaproject/Constants.java

```diff
@@ -0,0 +1,22 @@
+package com.cris.nvh.framgiaproject;
+
+/**
+ * Created by nvh
+ * Contact: toiyeuthethao1997@gmail.com
+ */
+
+public interface Constants {
+	public static final String CLIENT_ID = "&client_id=";
+	public static final String BASE_URL_GENRES = "https://api-v2.soundcloud.com/charts?kind=top&";
+	public static final String BASE_URL_TRACK = "http://api.soundcloud.com/tracks";
+	public static final String PARAMETER_LIMIT = "&limit=";
+	public static final String PARAMETER_OFFSET = "&offset=";
+	public static final String PARAMETER_SEARCH = "&q=";
+	public static final int LIMIT = 10;
+	public static final String ALL_MUSIC = "genre=soundcloud:genres:all-music";
+	public static final String ALL_AUDIO = "genre=soundcloud:genres:all-audio";
+	public static final String ALTERNATIVE = "genre=soundcloud:genres:salternativerock";
+	public static final String AMBIENT = "genre=soundcloud:genres:ambient";
+	public static final String CLASSICAL = "genre=soundcloud:genres:classical";
+	public static final String COUNTRY = "genre=soundcloud:genres:country";
```

Comment:
16 đến 21 em nên đưa ra thành một lớp Anotation và để ở package dùng nó.
https://developer.android.com/reference/android/support/annotation/StringDef
Không nên đưa vào Constants. sau sẽ phìng to.